### PR TITLE
[CI] Add missing value: property in issue template

### LINKED ISSUE #46: [CI] Add missing `value:` property in issue template
### What would you like to share?

Add `value:` property to the `attributes:` under `id: description` in [bug.yaml](https://github.com/Pradumnasaraf/LinkFree-CLI/blob/main/.github/ISSUE_TEMPLATE/bug.yaml) Issue template.

### Changes need to add

This complete thing needs to be added

```yaml
value: |
  ###  what is the problem?
  
  ###  what is your suggested solution?
```

### After changes

The template will look like this after the addition.

```yaml
  - type: textarea
    id: description
    attributes:
      label: Description
      description: A brief description of the question or issue, also include what you tried and what didn't work
      value: | # This will get added
        ###  What is the problem?
        
        ###  What is your suggested solution?
    validations:
      required: true
```

### PR DESCRIPTION
**Pull request by PiseAtharva**
## 🛠️ Fixes Issue
Example: Closes #46 

## 👨‍💻 Changes proposed
* Added value(text) for bug.yaml textarea
* includes problem definition and proposed solution requests

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -7,6 +7,9 @@ body:
     attributes:
       label: Description
       description: A brief description of the question or issue, also include what you tried and what didn't work
+      value: |
+        ### What is your problem
```
Comment (Pradumnasaraf) [inline .github/ISSUE_TEMPLATE/bug.yaml:11]:
```suggestion
        ### What is your problem
        
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

Thanks for the PR. Just and extra line needs to be added to make this PR ready to be merged.

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 1 above)
Comment (PiseAtharva) [reply .github/ISSUE_TEMPLATE/bug.yaml:11]: Thanks for the suggestion (cant believe I missed that), I committed that locally and pushed it into the feat/add branch if you dont mind

**Review by Ananya2001-an — APPROVED** [review]
Looks good to merge 👍

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**